### ISSUE TITLE
Remove all the warnings 

### ISSUE THREAD

**Issue by AtlasMaxima-zz** [QuantEcon/Bookshelf#360]
Remove all the warnings with the `href` such as `The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles.`
